simplerunner.lua

```diff
@@ -57,7 +57,7 @@ function run_single_benchmarks(benchname)
 end
 
 if arg[1] then
-    run_single_benchmarks(arg[1])
+    run_single_benchmarks(arg[1], arg[2])

```

Comment:
Is this recursive?

Reply:
ah i see, ignore me.

Reply:
what do you mean that's the optional run count it can be nil and default to 1 inside runbench

Reply:
ignore this comment. I misread the code.